src/app/FakeLib/MSBuildHelper.fs

```diff
@@ -16,29 +16,58 @@ exception BuildException of string*list<string>
   with
     override x.ToString() = x.Data0.ToString() + "\r\n" + (separated "\r\n" x.Data1)
 
+
+type MsBuildEntry = {
+    Version: string;
+    Paths: string list;
+}
+
+let knownMsBuildEntries = 
+    [
+        { Version = "12.0"; Paths = [@"\MSBuild\12.0\Bin"; @"\MSBuild\12.0\Bin\amd64"] }
+        { Version = "14.0"; Paths = [@"\MSBuild\14.0\Bin"] }
+        { Version = "15.0"; Paths = [@"\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin"; 
+                                     @"\Microsoft Visual Studio\2017\Professional\MSBuild\15.0\Bin";
+                                     @"\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin";
+                                     @"\MSBuild\15.0\Bin"] }
+    ]
+
+let oldMsBuildLocations = 
+    [ @"c:\Windows\Microsoft.NET\Framework\v4.0.30319\"; 
+      @"c:\Windows\Microsoft.NET\Framework\v4.0.30128\";
+      @"c:\Windows\Microsoft.NET\Framework\v3.5\"
+    ]    
+
+let toDict items = 
```

Comment:
can we call it toMap - since there is a dict function in FSharp.Core that actually put things in a dict